doc/python/sphinx/glossary.rst

```diff
@@ -27,3 +27,17 @@ Glossary
     the returned object doesn't have restrictions (i.e. ``None`` allowed). The
     deserializer is invoked with inbound message bytes on both the server side
     and the client-side.
+
+  wait_for_ready
+    If an RPC is issued but the channel is in TRANSIENT_FAILURE or SHUTDOWN
+    states, the RPC is unable to be transmitted promptly. By default, gRPC
```

Comment:
Suggestion: Prefer active voice to passive: "the library cannot transmit the RPC at that moment."

Reply:
Updated!